record/index.html.in

```diff
@@ -39,6 +39,9 @@
                         Related Records
                     </strong>
                 </a>&nbsp;&nbsp;
+                <!-- download CSV included by default -->
+                <export has-values="recordValues.length>0" reference="reference" style="position:relative;" disabled="!displayReady"></export>
```

Comment:
I think `has-values` must always be true for record app. We always want to show the button. 

`recordValues` might be empty (although that's not a normal case and it can only happen when the visible-columns is defined as an empty list) even though the data exists. Also we're going to show an error if the record doesn't exist so showing the button won't be harmful.